File: data/avro-schema/src/main/java/net/corda/data/chunking/PropertyKeys.java
package net.corda.data.chunking;

public final class PropertyKeys {
    private PropertyKeys() {
    }

    /**
     * Boolean property specifying whether same data content can re-upload again
     */
    public static final String FORCE_UPLOAD = "forceUpload";

    /**
     * Boolean property specifying whether the vnode databases should be deleted and recreated
     */
    public static final String RESET_DB = "resetDb";

    /**
     * String property specifying the RPC actor that made the request
     */
    public static final String ACTOR = "actor";
}

File: data/avro-schema/src/main/kotlin/net/corda/data/chunking/PropertyKeys.kt
package net.corda.data.chunking

object PropertyKeys {

    /**
     * Boolean property specifying whether same data content can re-upload again
     */
    const val FORCE_UPLOAD = "forceUpload"
    /**
     * Boolean property specifying whether the vnode databases should be deleted and recreated
     */
    const val RESET_DB = "resetDb"
    /**
     * String property specifying the RPC actor that made the request
     */
    const val ACTOR = "actor"
}
